components/strava/actions/update-activity/update-activity.mjs

```diff
@@ -0,0 +1,32 @@
+import strava from "../../strava.app.js";
+
+export default {
+  name: "Update Activity",
+  description: "Updates the given activity that is owned by the authenticated athlete. See `https://developers.strava.com/docs/reference/`",
```

Comment:
Could you please use []() to ref the docs?

Reply:
done